src/scss/includes/top_bar.scss

```diff
@@ -48,12 +21,9 @@
     top: 0;
     left: 0;
     width: 100%;
-    background: url('../images/backgrounds/top_mobile.svg') no-repeat;
```

Comment:
Can we delete this file too :smiley:

Reply:
We can't, it's still use for the mobile direction panel, which hasn't been redesigned yet.

Reply:
ah ok, let's be patient then